gatling-maven-plugin/src/main/java/io/gatling/mojo/GatlingMojo.java

```diff
@@ -223,6 +229,25 @@ public void execute() throws MojoExecutionException {
     }
   }
 
+  private void iterateBySimulations(Toolchain toolchain, List<String> jvmArgs, List<String> testClasspath, Collection<String> simulations) throws Exception {
+    Exception[] exc = new Exception[1];
+    for (String simulation : simulations) {
+      try {
+        executeGatling(jvmArgs, gatlingArgs(simulation), testClasspath, toolchain);
+      } catch (Exception e) {
+        if (continueAfterFail) {
+          exc[0] = e;
```

Comment:
It feels more natural to me to only buffer the first exception, instead of the first one.